test/test_mkldnn.py

```diff
@@ -192,6 +203,30 @@ def test_conv2d(self):
             self._test_serialization(mkldnn_conv2d, (x.to_mkldnn(),))
             self._test_tracing(mkldnn_conv2d, (x.to_mkldnn(),))
 
+    @unittest.skipIf(not has_bf16_support(), "OneDNN bfloat16 path requires AVX512")
```

Comment:
these tests should not be skipped if there's no bf16 support, instead they should be catching and asserting that appropriate errors are raised.

Reply:
Add test cases if there's no bf16 support.